modules/ppcp-button/resources/js/modules/ContextBootstrap/CheckoutBootstap.js

```diff
@@ -24,9 +24,11 @@ class CheckoutBootstap {
 
           })
 
-        jQuery('#saved-credit-card').on('change', () => {
-            this.displayPlaceOrderButtonForSavedCreditCards()
-        })
+        setTimeout(() => {
+            jQuery('#saved-credit-card').on('change', () => {
+                this.displayPlaceOrderButtonForSavedCreditCards()
+            })
+        }, 3000)
```

Comment:
What's the purpose of this delay?

Reply:
It seems that sometimes the listener is not added because the element is not loaded yet, that´s the reason of the delay here. I´m not a big fan of delays but I didn´t found a better way at the moment.

Reply:
Done.